dashboard/src/modules/components/OverviewComponent/SavedRunsComponent.jsx

```diff
@@ -66,18 +64,24 @@ const SavedRunsComponent = () => {
   const moreActionItems = (dataset) => [
     {
       title:
-        dataset.metadata[DATASET_ACCESS] === "public" ? "Unpublish" : "Publish",
+        dataset.metadata.dataset.access === "public" ? "Unpublish" : "Publish",
       onClick: () => {
         const accessType =
-          dataset.metadata[DATASET_ACCESS] === "public" ? "private" : "public";
+          dataset.metadata.dataset.access === "public" ? "private" : "public";
         dispatch(publishDataset(dataset, accessType));
       },
     },
     {
-      title: dataset.metadata[DASHBOARD_SEEN] ? "Mark unread" : "Mark read",
+      title: dataset.metadata.global.dashboard.seen
+        ? "Mark unread"
+        : "Mark read",
       onClick: () =>
         dispatch(
-          getMetaDataActions(dataset, "read", !dataset.metadata[DASHBOARD_SEEN])
+          getMetaDataActions(
+            dataset,
+            "read",
+            !dataset.metadata.global.dashboard.seen
```

Comment:
I thought `seen` and `saved` were more or less independent. Do we know for sure that `seen` has been defined in the "saved runs" view? Should this be chained conditional, to be safe?

Reply:
True